ci/prow/make_config.go

```diff
@@ -714,6 +714,7 @@ func generatePeriodic(title string, repoName string, periodicConfig yaml.MapSlic
 			}
 			jobNameSuffix = "webhook-apicoverage"
 			data.Base.Command = webhookAPICoverageScript
+			addEnvToJob(&data.Base, "SYSTEM_NAMESPACE", "knative-serving")
```

Comment:
This is hardcoded for serving, I believe this is per repo, right? So it would be:

```suggestion
			addEnvToJob(&data.Base, "SYSTEM_NAMESPACE", data.Base.RepoNameForJob)
```

Reply:
Fixed.